src/utilities/fswSetupRW.py

```diff
@@ -27,7 +27,7 @@
 def create(
         gsHat_B,
         Js,
-        uMax = numpy.NaN
+        uMax = numpy.nan
```

Comment:
This may be out of scope for this PR but it was necessary to run with the version of numpy I had installed.

Reply:
Are you sure? The docs suggest that nan, NaN, and NAN are aliases. 

https://numpy.org/doc/stable/reference/constants.html#numpy.nan

Reply:
Interesting, I will revert this change but for some reason my Linux python environment was throwing errors/warnings.